packages/apollo-links/src/TokenRefreshLink.js

```diff
@@ -150,10 +152,10 @@ export class TokenRefreshLink extends ApolloLink {
       }).subscribe({
         error: reject,
         next: ({ data }) => {
-          if (data === null || data.userRefreshToken === null) {
+          if (data === null || R.path(['system', 'userRefreshToken'], data) === null) {
             reject(new RefreshTokenInvalidError());
           } else {
-            const { refreshToken, idToken }: RefreshTokenQueryResult = data.userRefreshToken;
+            const { refreshToken, idToken }: RefreshTokenQueryResult = R.pathOr({}, ['system', 'userRefreshToken'], data);
```

Comment:
I think you should move path `['system', 'userRefreshToken']` to constant and share between [155](https://github.com/8base/sdk/pull/20/files#diff-b2d264331219a298f27a25635ac735bfR155) and [158](https://github.com/8base/sdk/pull/20/files#diff-b2d264331219a298f27a25635ac735bfR158)